### DISCUSSION TITLE
Could anyone help to review our app from v1 -> v2, it was always pending review status

### DISCUSSION THREAD

**Discussion by anshuomomo** [WalletConnect/walletconnect-monorepo#4282] [Developer Support] [answered]
Hi, 
     We have integrated walletconnect v2 for a long time, and found that the link in W3bmodal seems still v1, and we found that we need to change the setting in the could.walletconnect.com and we have submitted a review last week, but it is always in PENDING REVIEW. Could some one help with this? That will be very helpful!
    Our app project ID: 4202cfb72696dde11177cd3172ea9f49
    Best Regards.

**Comment by glitch-txs** [comment]
We won't be accepting submission in the near future. This doesn't affect WalletConnect functionally.

Do you have any issue in the integration?

**Reply by anshuomomo** [reply]
Hi, @glitch-txs Hope you are doing well.

My problem is this:
If user search and choose our app( project ID: 4202cfb72696dde11177cd3172ea9f49)  in the Web3modal, is still a v1 link, but that will be not work to connect with our wallet, we have supported v2.
 
Could you please help to change this setting?

**Reply by glitch-txs** [reply]
Which SDK are you using?

If you're using Web3Modal make sure to be using v4
https://docs.walletconnect.com/web3modal/about

**Reply by anshuomomo** [reply]
I think it is maybe not the problem of the verision of the web3modal. We are not developing a dapp with Web3Modal, we are the wallet end. We have received some dapps developer's feedback that they can not connect to our wallet using web3modal. I found the link is:  `wc:d53415ee-83bf-483a-9bdc-7bf5732c15bf@1?bridge=https%3A%2F%2Fy.bridge.walletconnect.org&key=7056a8eedc64856edd9e347b0cd7f4cd8425d417171242b62fdaae45182f8f8f`
<img width="439" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/16026265/93d1a831-d65c-4d1f-8ca9-a285276682d3">

And we are now only support link v2, like: `wc:4a2d33d41563ba99b32a2bf33fe6c13a6623cc0ac06c4ed3ed2d01ad3855647e@2?relay-protocol=irn&symKey=a701dd3a0e67876758782d239dcc7e22ee426f8c85a92b6365d26d692440ddee`

I think the config with our wallet in the web3modal should be updated,  I tried to submit a review here:
<img width="1109" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/16026265/ba085a6c-abb6-44b5-9559-e91299716c3b">
on https://cloud.walletconnect.com/app but is not work.

Is there anyway else where i can update this option?

**Reply by glitch-txs** [reply]
appologies, I thought it was for a Dapp, we are taking submitions for wallets. The Cloud approval will make your wallet show up in Web3Modal and will be included in the explorer API, however you should be able to connect with QR code without it.

We tested the wallet and were not able to connect. Could you please test your wallet here: https://evmtest.walletconnect.com/
It should properly work on iOS and Android. You can also add your wallet's data there to test your deeplink.

Some additional relevant information:
- Web3Wallet documentation: https://docs.walletconnect.com/web3wallet/about
- Namespaces support: https://docs.walletconnect.com/advanced/migration-from-v1.x/namespaces

Feel free to resubmit when this is addressed. Thanks!

**Reply by anshuomomo** [reply]
@glitch-txs  OK, thanks.  We will test and fix ASAP.

**Reply by anshuomomo** [reply]
Hi  @glitch-txs , 
We have published new version both on iOS(v3.0.3) and on Android Google Play (v3.0.3) to fix our wallet([download here](https://iopay.me/)). Would you please help to review that and re-published our wallet in your project?

**Reply by glitch-txs** [reply, marked as answer]
flawless! Thanks so much! Your wallet was approved and published.
